dev/io.openliberty.webcontainer.servlet.6.1.internal_fat/publish/servers/servlet61_HttpResponseCodesTest/bootstrap.properties

```diff
@@ -0,0 +1,15 @@
+###############################################################################
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+bootstrap.include=../testports.properties
+osgi.console=7777
+com.ibm.ws.logging.trace.specification=*=info:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:GenericBNF=all:io.openliberty.webcontainer*=all
```

Comment:
For a test of this nature, where we are really just testing some constants in the API is it really necessary to generate all  of this trace?